domene/src/main/kotlin/no/nav/tiltakspenger/saksbehandling/domene/behandling/BehandlingTilstand.kt

```diff
@@ -1,5 +1,5 @@
 package no.nav.tiltakspenger.saksbehandling.domene.behandling
 
 enum class BehandlingTilstand {
-    OPPRETTET, VILKÅRSVURDERT, TIL_BESLUTTER, IVERKSATT
+    OPPRETTET, UNDER_BEHANDLING, TIL_BESLUTTER, IVERKSATT
```

Comment:
OPPRETTET skal vel slettes? Tilstand og Status merges? Kan godt gjøre i neste PR

Reply:
For da tenker du at vi sletter hele behandlingen når en SBH fjernes fra behandlingen? (fremtidig funksjonalitet)

Reply:
Nei. Men da bør vi rename den til KLAR_TIL_BEHANDLING. Dvs bruke statusene fra behandlingBenkStatuser. Men vi kan godt ha det som en todo i denne PRen